src/main.rs

```diff
@@ -131,8 +135,8 @@ async fn get_lnurl_struct(
     State(state): State<ClnurlState>,
 ) -> Result<Json<LnurlResponse>, StatusCode> {
     Ok(Json(LnurlResponse {
-        min_sendable: AmountWrapper::from_msat(1),
-        max_sendable: AmountWrapper::from_msat(100000000000),
+        min_sendable: Amount::from_msat(100),
```

Comment:
Should probably be configurable instead of tinkering with it in the code.

Reply:
#10